### PR TITLE
Expand on application sandboxing

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: Develop/Apps/Application_Permissions/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+[X-Sailjail]
+Permissions=Camera;Internet;Pictures
+OrganizationName=org.foobar
+ApplicationName=MyApp
+```
+
+The complete listing of all available application permissions together with more detailed information on configuring the sandbox for your applications can be found in the [sailjail-permissions](https://github.com/sailfishos/sailjail-permissions) repository. List of permissions available to applications ditributed through [Harbour](/Develop/Apps/Harbour) can be found from the [Harbour FAQ](https://harbour.jolla.com/faq#4.7.0).
+
+See the [Migrating configuration and data files for sandboxed apps](https://forum.sailfishos.org/t/migrating-configuration-and-data-files-for-sandboxed-apps/8866) post on Forum to learn how to adapt your existing applications for the transition to sandboxed execution.
```
[the remark this answers is not in the document]
Comment (martyone) [reply Develop/Apps/Application_Permissions/README.md]: Added that information to the sailjail-permissions repo linked from here (with https://github.com/sailfishos/sailjail-permissions/pull/125) and dropped the link to forum here.

File: Reference/Security/README.md [hunk 2]
```diff
@@ -15,12 +15,10 @@ Privacy is similarly important. Privacy is related to the way that consumer data
...
+Sailfish OS currently uses a three-level security architecture.
 
-Linux user groups are used to separate privileged applications from non-privileged applications, and file system access is enforced by the Linux kernel.
-
-### Future Security Architecture
-
-In the future, Sailfish OS will use application sandboxing and per-application access control lists to control application behaviour and limit the scope of malicious activity achievable by exploiting a vulnerability in any given application or service. This security architecture will be applied to core Sailfish OS services and applications, as well as third-party Harbour applications.
+- Third-party applications submitted through the [Harbour](/Develop/Apps/Harbour) are rigorously examined for any signs of malicious behaviour.
+- Every application available from [Store](/Services/Deployment/Store), irrespective of its origin, is run in a [Sailjail](/Develop/Apps/Application_Permissions) sandbox with an explicitly assigned set of application permissions to limit the scope of malicious activity achievable by exploiting a possible vulnerability in the application.
```
Comment (rainemak) [inline Reference/Security/README.md:23]: This is not only `Store apps` rather all application are sandboxed by default and default profile is applied for them when permissions are not explicitly defined. See https://github.com/sailfishos/sailjail-permissions#sandboxing-of-applications-without-application-profile

**Review by rainemak — APPROVED** [review]

File: Develop/Apps/Application_Permissions/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Permissions=Camera;Internet;Pictures
+OrganizationName=org.foobar
+ApplicationName=MyApp
+```
+
+The complete listing of all available application permissions together with more detailed information on configuring the sandbox for your applications can be found in the [sailjail-permissions](https://github.com/sailfishos/sailjail-permissions) repository. It covers topics related to both newly created applications and applications that were created before sandboxing was introduced, in which case an additional effort is needed in order to migrate application data and configuration to the new sandboxed locations.
+
+List of permissions available to applications distributed through [Harbour](/Develop/Apps/Harbour) can be found from the [Harbour FAQ](https://harbour.jolla.com/faq#4.7.0).
+
+Be aware that during development, when an application is run from the Sailfish IDE, from terminal or by similar means, sandboxing is not active and the application runs without restrictions.
```
Comment (Thaodan) [inline Develop/Apps/Application_Permissions/README.md:32]: Can you mention that:
- Applications not started from the launcher aren't sandboxed (or at least not with Sailjail).
- Applications can be started through `sailjail /usr/bin/harbour-foobar` during development.

Comment (rainemak) [reply Develop/Apps/Application_Permissions/README.md:32]: > * Applications not started from the launcher aren't sandboxed (or at least not with Sailjail).

I'd not needed. I think terminal is enough here. Let's do not add too much emphasis on how to not to run apps without sandbox. Instead, let's try to help and find ways and best practices to work with sandboxing.

> * Applications can be started through sailjail /usr/bin/harbour-foobar during development.

Yes, this is a good addition indeed. How to debug sandboxing could be linked against [this](https://github.com/sailfishos/sailjail/blob/master/APPDEBUG.md).

Comment (martyone) [reply Develop/Apps/Application_Permissions/README.md]: Updated

**Review by rainemak — APPROVED** [review]

**Comment by martyone** [comment]
Rebased

**Pull request merged**